features/karaf/openhab-tp/src/main/feature/feature.xml

```diff
@@ -179,6 +179,12 @@
 		<feature dependency="true">openhab.tp-xtext</feature>
 	</feature>
 
+	<feature name="openhab.tp-micrometer" description="Micrometer instrumentation library" version="${project.version}">
+		<capability>openhab.tp;feature=micrometer;version=1.6.2</capability>
+		<bundle>wrap:mvn:io.micrometer/micrometer-core/1.6.2$Bundle-SymbolicName=micrometer-core&amp;Bundle-Version=1.6.2</bundle>
```

Comment:
Using the wrap protocol causes issues when for instance debugging the code in non-Karaf environments. E.g. it won't work when running the Demo App in Eclipse. To fix that you can either use bnd or Maven to embed a non-OSGi dependency into your bundle. See for instance the `org.openhab.core.audio` bundle for embedding a dependency in your bundle using Maven.